tutorials/3d/procedural_geometry/surfacetool.rst

```diff
@@ -79,6 +116,21 @@ to remove duplicate vertices.
     # Alternatively:
     st.index()
 
+ .. code-tab:: csharp
+
+    // Creates a quad from four corner vertices.
+    // Add_index does not need to be called before add_vertex.
```

Comment:
```suggestion
    // AddIndex does not need to be called before AddVertex.
```
Could you also change the GDScript comment to say `add_index` as it shouldn't have a capital letter